### DISCUSSION TITLE
salvataggio impostazioni

### DISCUSSION THREAD

**Discussion by MicCosta** [arkypita/LaserGRBL#1947] [General]
Salve  dopo aver  caricato un immagine e averla  lavorata è possibile salvare  questa impostazione per replicarla senza dover creare un databese  esterno?
grazie

**Comment by arkypita** [comment]
No, attualmente questa prestazione non c'è e non è facile da aggiungere per come è fatto il software.
Puoi invece usare il database materiali per salvare coppie velocità/potenza per i diversi materiali in base ai tuoi test, basta aggiungere nuove righe in fondo.
